src/qmflows/packages/serializer.py

```diff
@@ -87,3 +95,27 @@ def encode(self, obj: NDFrame, make_rec: Callable[[T], Dict[str, T]]) -> Dict[st
     def decode(self, cls: Type[NDFrame], data: Mapping) -> NDFrame:
         """Decode the passed data into a pandas Series or DataFrame."""
         return cls(data)
+
+
+class SerReduce(Serialiser):
```

Comment:
So, we need this new `Serialiser` to make **pickle** happy?

Reply:
More specifically to make `noodles` happy; `pickle` is already content with `Package.__reduce__`.

Reply:
Nice to know :)